src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2119,6 +2119,8 @@ internal enum ErrorCode
 
         #region diagnostics introduced for C# 12.0
         ERR_ImplicitlyTypedDefaultParameter = 9066,
+        ERR_OptionalParamValueMismatch = 9067,
+        WRN_OptionalRequiredParamMismatch = 9068
```

Comment:
>WRN_OptionalRequiredParamMismatch [](http://example.com/codeflow?start=8&length=33)

Is this warning being reported currently?

Reply:
No - just removed it